src/src/common.h

```diff
@@ -50,43 +58,69 @@ typedef enum
 
 typedef enum
 {
-    RATE_200HZ = 0,
-    RATE_100HZ = 1,
-    RATE_50HZ = 2,
-    RATE_25HZ = 3,
-    RATE_4HZ = 4
+    RATE_500HZ = 0,
+    RATE_250HZ = 1,
+    RATE_200HZ = 2,
+    RATE_150HZ = 3,
+    RATE_100HZ = 4,
+    RATE_50HZ = 5,
+    RATE_25HZ = 6,
+    RATE_4HZ = 7
 } expresslrs_RFrates_e; // Max value of 16 since only 4 bits have been assigned in the sync package.
 
-#define RATE_MAX 5
+typedef struct expresslrs_rf_pref_params_s
+{
+    int8_t index;
+    expresslrs_RFrates_e enum_rate; // Max value of 16 since only 4 bits have been assigned in the sync package.
+    int32_t RXsensitivity;          //expected RF sensitivity based on
+    uint32_t TOA;                   //time on air in microseconds
+    uint32_t RFmodeCycleInterval;
+    uint32_t RFmodeCycleAddtionalTime;
+    uint32_t SyncPktIntervalDisconnected;
+    uint32_t SyncPktIntervalConnected;
+
+} expresslrs_rf_pref_params_s;
 
+#if defined(Regulatory_Domain_AU_915) || defined(Regulatory_Domain_EU_868) || defined(Regulatory_Domain_FCC_915) || defined(Regulatory_Domain_AU_433) || defined(Regulatory_Domain_EU_433)
+#define RATE_MAX 5
+#define RATE_DEFAULT 0
```

Comment:
Does this try to set the default to RATE_500HZ = 0?

Reply:
No, it will be 200hz, It's the first index of the table, not the first enum